api/rest/swagger.yaml

```diff
@@ -0,0 +1,796 @@
+openapi: 3.0.1
+info:
+  title: IPFS Cluster API
+  description: HTTP API to interact with IPFS Cluster.
+  version: "0.10.1"
+servers:
+  - url: 'http://127.0.0.1:9094'
+paths:
+  /id:
+    get:
+      summary: Get ID of Cluster
+      tags:
+        - cluster
+      description: Get the ID of the Cluster and it's associated IPFS node.
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Peer'
+              example: {"id":"QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","addresses":["/p2p-circuit/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","/ip4/127.0.0.1/tcp/9096/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","/ip4/192.168.86.35/tcp/9096/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","/ip4/10.6.0.2/tcp/9096/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","/ip4/192.168.1.42/tcp/9096/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH"],"cluster_peers":["QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH"],"cluster_peers_addresses":null,"version":"0.10.0","commit":"","rpc_protocol_version":"/ipfscluster/0.10/rpc","error":"","ipfs":{"id":"QmV8LYMZ1KLRzJacFCXrUjYJdHN4XcdTvLmswdkzwVjfse","addresses":["/ip4/127.0.0.1/tcp/4001/ipfs/QmV8LYMZ1KLRzJacFCXrUjYJdHN4XcdTvLmswdkzwVjfse","/ip4/192.168.86.35/tcp/4001/ipfs/QmV8LYMZ1KLRzJacFCXrUjYJdHN4XcdTvLmswdkzwVjfse","/ip4/10.0.3.1/tcp/4001/ipfs/QmV8LYMZ1KLRzJacFCXrUjYJdHN4XcdTvLmswdkzwVjfse","/ip4/10.6.0.2/tcp/4001/ipfs/QmV8LYMZ1KLRzJacFCXrUjYJdHN4XcdTvLmswdkzwVjfse","/ip4/192.168.1.42/tcp/4001/ipfs/QmV8LYMZ1KLRzJacFCXrUjYJdHN4XcdTvLmswdkzwVjfse","/ip6/::1/tcp/4001/ipfs/QmV8LYMZ1KLRzJacFCXrUjYJdHN4XcdTvLmswdkzwVjfse"],"error":""},"peername":"omou"}
+  /version:
+    get:
+      summary: Get Version of Cluster
+      tags:
+        - cluster
+      description: Get the current version of the IPFS Cluster.
```

Comment:
`This endpoint returns the version of the IPFS Cluster daemon responding to the request.`